dnf-behave-tests/dnf/plugins-core/config-manager-opts.feature

```diff
@@ -0,0 +1,371 @@
+   When I execute dnf with args "config-manager setopt repo1.skip_if_unavailable=1 --create-missing-dir"
+   Then the exit code is 0
+    And file "/etc/dnf/repos.override.d/99-config_manager.repo" contents is 
+      """
+      # Generated by dnf5 config-manager.
+      # Do not modify this file manually, use dnf5 config-manager instead.
+      [repo1]
+      skip_if_unavailable=1
+      """
+
+
+Scenario: repository configuration overrides - new option, a file was found instead of destination directory
+ Given I create file "/etc/dnf/repos.override.d" with
+      """
+      """
+   When I execute dnf with args "config-manager setopt repo1.skip_if_unavailable=1"
+   Then the exit code is 1
+    And stderr contains "The path ".*/etc/dnf/repos.override.d" exists, but it is not a directory or a symlink to a directory"
+
+
+Scenario: repository configuration overrides - new option, a symlink to non-existent object was found instead of destination directory
+ Given I create symlink "/etc/dnf/repos.override.d" to file "/non-exist"
+   When I execute dnf with args "config-manager setopt repo1.skip_if_unavailable=1"
+   Then the exit code is 1
+    And stderr contains "The path ".*/etc/dnf/repos.override.d" exists, but it is a symlink to a non-existent object"
+
+
+Scenario: repository configuration overrides - unset/remove option
+ Given I create file "/etc/dnf/repos.override.d/99-config_manager.repo" with
+      """
+      [repo1]
+      enabled=0
+      priority=40
+      skip_if_unavailable=1
+      [repo2]
+      enabled=1
+      priority=50
+      skip_if_unavailable=0
+      """
+   When I execute dnf with args "config-manager unsetopt repo1.priority"
+   Then the exit code is 0
+    And file "/etc/dnf/repos.override.d/99-config_manager.repo" contents is
+      """
+      [repo1]
+      enabled=0
+      skip_if_unavailable=1
+      [repo2]
+      enabled=1
+      priority=50
+      skip_if_unavailable=0
+      """
+
+
+Scenario: repository configuration overrides - unset/remove option, globs in repo id
+ Given I create file "/etc/dnf/repos.override.d/99-config_manager.repo" with
+      """
+      [repo1]
+      enabled=0
+      priority=40
+      skip_if_unavailable=1
+      [repo2]
+      enabled=1
+      priority=50
+      skip_if_unavailable=0
+      [test_repo]
+      enabled=1
+      priotity=80
+      skip_if_unavailable=0
+      """
+   When I execute dnf with args "config-manager unsetopt r*o?.priority"
+   Then the exit code is 0
+    And file "/etc/dnf/repos.override.d/99-config_manager.repo" contents is
+      """
+      [repo1]
+      enabled=0
+      skip_if_unavailable=1
+      [repo2]
+      enabled=1
+      skip_if_unavailable=0
+      [test_repo]
+      enabled=1
+      priotity=80
+      skip_if_unavailable=0
+      """
+
+
+Scenario: repository configuration overrides - unset/remove option, trying to unset an not set option is OK
```

Comment:
I think a warning for repository use case is essential. When I have a repository in yum.repos.d and I would like to modify the file by removing an option, config manager is not fulfill the requirement silently.